extensions/publisher-command-center/src/components/ContentsComponent.js

```diff
@@ -83,6 +109,7 @@ const ContentsComponent = {
                 m("a", {
                   class: "fa-solid fa-arrow-up-right-from-square",
                   href: content["content_url"],
+                  title: `Open ${title} (opens in new tab)`,
```

Comment:
```suggestion
                  title: `Open ${title}`,
```
The "(opens in a new tab)" feels a bit verbose to me, but I'm not that adamant about removing it.

Reply:
I can move the "(opens in new tab)" portion to the `aria-label` instead of the tooltip. In general, if a link opens a new window or tab, the screen reader should alert the user of the action.

Reply:
> In general, if a link opens a new window or tab, the screen reader should alert the user of the action.

Do screen readers not give that information when `target="_blank"`? I guess they don't

Reply:
Added `aria-label` in commit [3e93e62](https://github.com/posit-dev/connect-extensions/pull/138/commits/3e93e62329ed387b387cfad8473152180f5fe1c9).